cmd/agent/install_script.sh

```diff
@@ -267,7 +267,7 @@ if [ ! "$apikey" ]; then
   fi
 fi
 
-if [[ `uname -m` == "armv7l" ]] && [[ $agent_flavor == "datadog-agent" ]]; then
+if [[ $(uname -m) == "armv7l" ]] && [[ $agent_flavor == "datadog-agent" ]]; then
```

Comment:
CircleCI only checks some of the files, in specific `./cmd/**/*.sh` and `./omnibus/package-scripts/*/*`.

"But that should cover the install scripts!" - you will say.

And you will be right, however shellcheck allows using `` `<command>` ``  instead of `$(<command>)` if the shell type is set to `bash`. I didn't specify the shell type, so it changed those.

I'll revert the changes to the install scripts here as you suggested, so this PR should only change files that aren't checked by CircleCI.

Reply:
That makes sense, thanks for explaining!

Reply:
I wouldn't be against shell-linting checking everything to be honest